elastic/kibana
# [XY] Defaults the point size to 1 and corrects the vislib migrations

## Summary

This PR introduces the following changes:
- Changes the default point size to be 1 instead of 3. This means that all the new visualizations will default to a smaller point size

**Before**
![image](https://user-images.githubusercontent.com/17003240/142378918-b62a3d5e-63b0-4bbd-9a46-e88318bdd0c2.png)

**Now**
![image](https://user-images.githubusercontent.com/17003240/142378981-a56063c0-d0d9-4fdf-a764-64b7205437e2.png)

- Updates the xy migration script in order to add `ciclesRadius` attribute to 1 for all these visualizations that don't have the attribute (these will be visualizations made by vislib). All the visualizations that have been created with the new implementation have already this attribute and we don't change them.
- Updates the migration script to add as a fitting function the linear for all these visualizations that haven't this attribute.

### How to test
1. Create a new area/line visualization and check that the point size is 1
2. Create a visualization with vislib (for example from 7.11) and import it. The point size should be 1
3. Create a visualization with EC (for example from 7.15) and import it. The point size should be 3.

Here is a screenshot with a 7.11 area imported to this branch. The point size is set to 1
![image](https://user-images.githubusercontent.com/17003240/142379513-e7d2eeb4-2cc6-4528-88b3-a325e27d72dd.png)



### Checklist
